apps/aeutils/src/aeu_env.erl

```diff
@@ -559,17 +582,30 @@ lst(L) when is_list(L) -> L;
 lst(E) -> [E].
 
 update_config(Map) when is_map(Map) ->
+    update_config(Map, _Notify = true).
```

Comment:
does this generally mean we can update some or all configs also in runtime ?

Reply:
Generally, it sorta does, but also generally, not all code in the system may be prepared for runtime changes of the user config, so your mileage may vary at this time.